portal-web/test/functional/com/liferay/portalweb/macros/headless/headlessdeliveryapi/LanguagesAPI.macro

```diff
@@ -0,0 +1,43 @@
+definition {
+
+	macro _getAssetLibraryLanguages {
+		Variables.assertDefined(parameterList = "${assetLibraryId}");
```

Comment:
The methods look good, I'd like to share a small suggestion that we also can use "api" as a parameter in the future to reduce some duplicate codes:
if (isSet(assetLibraryId)){
  var api ="";
}else{ var api ="";}